tests/modules/images/dominant-color/dominant-color-test.php

```diff
@@ -0,0 +1,115 @@
+<?php
+/**
+ * Tests for dominant-color module.
+ *
+ * @package performance-lab
+ */
+class Dominant_Color_Test extends WP_UnitTestCase {
+
+	/**
+	 * Tests dominant_color_metadata().
+	 *
+	 * @dataProvider provider_set_of_images
+	 */
+	public function test_dominant_color_metadata( $image_path, $expected_color, $expected_transparency, $color_is_light ) {
+		// Non existing attachment.
+		$dominant_color_metadata = ( new WP_Dominant_Color() )->dominant_color_metadata( array(), 1 );
```

Comment:
There doesn't need to be multiple new instances of the same class in a test. You can create one instance and then reuse it.